web/src/containers/activity/Description.js

```diff
@@ -22,8 +22,8 @@ const Description = props => {
             name={`activity-summary-${activity.key}`}
             label="activity summary"
             hideLabel
-            className="m0 textarea"
-            rows="5"
+            className="m0 textarea textarea-sm"
+            rows="3"
```

Comment:
I tried to add the approximate number of rows equivalent to the CSS-specified height, just because it's something the HTML can render on its own so it's extra-bulletproof.